MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+{
+    "_comment": "General",
+    "itemGroup.integrateddynamics": "整合動力",
+    "general.integrateddynamics.description": "完全掌握並自動化你的裝置",
+    "general.integrateddynamics.info.none": "無",
+    "general.integrateddynamics.energy": "能量",
+    "general.integrateddynamics.energy_unit": "RF",
+    "general.integrateddynamics.error.no_network": "找不到網路",
+    "general.integrateddynamics.true": "True",
+    "general.integrateddynamics.false": "False",
+
+    "_comment": "Gui",
+    "gui.integrateddynamics.part_settings": "設備設定",
+    "gui.integrateddynamics.aspect_settings": "項目設定",
+    "gui.integrateddynamics.part_offsets": "設備偏移量",
+    "gui.integrateddynamics.keybinding": "綁定鍵",
+    "gui.integrateddynamics.button.save": "保存",
+    "gui.integrateddynamics.button.rename": "重新命名",
+    "gui.integrateddynamics.button.copy": "複製",
+    "gui.integrateddynamics.button.copy.info": "複製到剪貼簿",
+    "gui.integrateddynamics.button.edit": "編輯",
+    "gui.integrateddynamics.button.add": "添加",
+    "gui.integrateddynamics.button.remove": "移除",
+    "gui.integrateddynamics.partsettings.update_interval": "每次間隔刻數",
+    "gui.integrateddynamics.partsettings.priority": "優先度",
+    "gui.integrateddynamics.partsettings.channel": "能量頻道",
+    "gui.integrateddynamics.partsettings.channel.disabledinfo": "Network energy consumption is disabled on this server.",
+    "gui.integrateddynamics.partoffset.offsets": "和目標位置的相對偏移量",
+    "gui.integrateddynamics.partoffset.offsets.max": "最大偏移量：%s",
+    "gui.integrateddynamics.partoffset.offsets.max.howtoincrease": "用設備強化：偏移量來增加最大值",
+    "gui.integrateddynamics.partoffset.slot.message.noint": "無效的整數座標",
+    "gui.integrateddynamics.partoffset.slot.message.outofrange": "偏移量超過最大值",
+    "gui.integrateddynamics.partoffset.x": "X 偏移量",
+    "gui.integrateddynamics.partoffset.y": "Y 偏移量",
+    "gui.integrateddynamics.partoffset.z": "Z 偏移量",
+    "gui.integrateddynamics.partsettings.side": "目標方向",
+    "gui.integrateddynamics.delay.capacity": "容量",
+    "gui.integrateddynamics.part.properties": "Properties:",
+    "gui.integrateddynamics.logicprogrammer.filter": "過濾器",
+    "gui.integrateddynamics.rewards": "獎勵",
+    "gui.integrateddynamics.rewards.collect": "領取",
+    "gui.integrateddynamics.rewards.collected": "已領取",
+    "gui.integrateddynamics.rewards.collectFailure": "無法領取獎勵，也許是背包太滿了。",
+    "gui.integrateddynamics.input": "輸入：%s",
+    "gui.integrateddynamics.output": "輸出：%s",
+    "gui.integrateddynamics.mechanical_squeezer.fluidautoeject": "自動輸出液體",
```

Liquid = 液體
Fluid = 流體
請將所有的 `液體` 一鍵替換為 `流體`
```suggestion
    "gui.integrateddynamics.mechanical_squeezer.fluidautoeject": "自動輸出流體",
```